### PR TITLE
fix(copilot): Limit number of action buttons, change PR url

### PR DESCRIPTION
**Pull request by roaga**
Limits actions on suspect PR comment to 3. Also changes the url written in the comment to use the right number.

### CODE REVIEW

**Comment by sentaur-athena** [comment]
@roaga can you please provide more context so we can review easier? What's the goal and why? Why are we limiting the number to be 3?

**Comment by roaga** [comment]
> @roaga can you please provide more context so we can review easier? What's the goal and why? Why are we limiting the number to be 3?

@sentaur-athena We're limiting the number of actions to 3 because GitHub is telling us they don't want more than 3 to avoid cluttering the UI (since each action is rendered as a button on the comment). So this just handles those cases where a PR has more than 3 suspect issues.

And we're changing the number in the text payload to align with the number expected by Seer in the Autofix side.

File: src/sentry/integrations/github/tasks/pr_comment.py [hunk 1]
```diff
@@ -224,9 +224,9 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
             {
                 "name": f"Root cause #{i + 1}",
                 "type": "copilot-chat",
-                "prompt": f"@sentry root cause issue {str(issue_id)} with PR URL https://github.com/{repo.name}/pull/{str(pullrequest_id)}",
```
Comment (sentaur-athena) [inline src/sentry/integrations/github/tasks/pr_comment.py:227]: Is this currently broken in prod?

File: src/sentry/integrations/github/tasks/pr_comment.py [hunk 2]
```diff
@@ -224,9 +224,9 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
...
+                "prompt": f"@sentry root cause issue {str(issue_id)} with PR URL https://github.com/{repo.name}/pull/{str(pr_key)}",
             }
-            for i, issue_id in enumerate(top_24_issues)
+            for i, issue_id in enumerate(top_24_issues[:3])
```
Comment (sentaur-athena) [inline src/sentry/integrations/github/tasks/pr_comment.py:229]: Should we say something about the fact that there can be more that we're not showing here?

Comment (roaga) [reply src/sentry/integrations/github/tasks/pr_comment.py:229]: GitHub doesn't really give us a mechanism to do that. The comment body itself shows all the issues already.

File: src/sentry/integrations/github/tasks/pr_comment.py (hunk 1 above)
Comment (roaga) [reply src/sentry/integrations/github/tasks/pr_comment.py:227]: It doesn't break anything on the Sentry side. Our logic in Seer just wants a different number. And this isn't available to any customers.

**Review by sentaur-athena — APPROVED** [review]

**Pull request merged**